samples/subsys/testsuite/pytest/shell/testcase.yaml

```diff
@@ -33,3 +33,13 @@ tests:
       - CONFIG_SHELL_VT100_COLORS=n
     harness_config:
       shell_commands: *kernel_commands
+  sample.pytest.required_by_other_app:
```

Comment:
is this change needed in this sample? if not, do not put something like this in an unrelated sample please, this now aimlessly builds more things than needed to verify or show how this unrelated feature work.
Maybe create something dedicated with filters to limit builds.

Reply:
ok, removed. It was not needed. I wanted only to show, that some configuration can be added with `build_only: true` in the same yaml, and it is easy to use it by other testsuite.